app/controllers/posts_controller.rb

```diff
@@ -264,7 +264,7 @@ def search
     if params[:subject].present?
       @search_results = @search_results.search(params[:subject]).where('LOWER(subject) LIKE ?', "%#{params[:subject].downcase}%")
     end
-    @search_results = @search_results.where(status: Post::Status::COMPLETE) if params[:completed].present?
+    @search_results = @search_results.complete if params[:completed].present?
```

Comment:
...did you make _scopes_ for the statuses?

Reply:
It looks like no unless there is Rails Magic going on here, this looks like it will do... something weird...

Reply:
There is Rails Magic that makes scopes for the statuses